feedbacks gerais.md

```diff
@@ -0,0 +1,35 @@
+Opa... passando pra te dar os feedbacks:
```

Comment:
Arquivos MD servem para documentar coisas, sempre que for documentar algo no projeto prefira arquivos nesse formato.. para ver ele na versão final clica no ícone de 📄  aqui em cima

Reply:
[Pode ver ele melhor nesse link também](https://github.com/AndreAntunesVieira/api-nasa/blob/feedbacks/feedbacks%20gerais.md)

Reply:
Certo de fato deveria ter instruído a inicialização do projeto no README.md